src/app/onboarding/components/glam/archive-creation-start-screen/archive-creation-start-screen.component.html

```diff
@@ -6,18 +6,23 @@
   <p class="greeting">Welcome to Permanent!</p>
 </div>
 <div class="text-buttons-container">
-  <div class="text">
-    <p>We’re so glad you’re here!</p>
-    <p>
-      At Permanent, it is our mission to provide a safe and secure place to
-      store, preserve, and share the digital legacy of all people, whether
-      that's for you or for your friends, family, interests or organizations.
-    </p>
-    <p>
-      We know that starting this journey can sometimes be overwhelming, but
-      don’t worry. We’re here to help you every step of the way.
-    </p>
-  </div>
+  @if (!hasShareToken) {
+    <div class="text text-no-share">
+      <p>We’re so glad you’re here!</p>
+      <p>
+        At Permanent, it is our mission to provide a safe and secure place to
+        store, preserve, and share the digital legacy of all people, whether
+        that's for you or for your friends, family, interests or organizations.
+      </p>
+      <p>
+        We know that starting this journey can sometimes be overwhelming, but
+        don’t worry. We’re here to help you every step of the way.
+      </p>
+    </div>
+  }
+  @if (hasShareToken) {
```

Comment:
I believe we can just use `@else` here. (I know, it's weird to be able to use an `else` statement in an Angular template!)